src/components/Comments/ListComment.js

```diff
@@ -49,6 +51,7 @@ const ListComment = ({
         </View>
         {item?.children_counts?.comment > 0 && (
           <ReplyComment
+            feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that the `feedId` prop is properly validated and used within the `ReplyComment` component. 

```diff
<ReplyComment
+  feedId={feedId}
   hideLeftConnector={hideLeftConnector(index, item, comments)}
   data={item.latest_children.comment}
   countComment={item.children_counts.comment}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->